documentation/asciidoc/accessories/camera/camera_hardware.adoc

```diff
@@ -269,3 +269,71 @@ Other available schematics;
 * Camera Module v3 https://datasheets.raspberrypi.com/camera/camera-module-3-schematics.pdf[PDF]
 * HQ Camera Module https://datasheets.raspberrypi.com/hq-camera/hq-camera-schematics.pdf[PDF]
 
+==== Camera Connector Pinning (15-Pin)
+
+This is the pinout of the 15-Pin Raspberry Pi CSI camera connector as used in the classical Rasberry Pi products. The connector is an Amphenol SFW15R-2STE1LF or compatible type.
```

Comment:
I'm not sure if "classical" is the right phrasing here. Looking at the wording on  https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#introduction perhaps it ought to be something like "flagship Raspberry Pi models before the Raspberry Pi 5".